client/extensions/woocommerce/woocommerce-services/state/packages/selectors.js

```diff
@@ -89,12 +92,19 @@ export const getAllSelectedPackages = createSelector(
  * @returns {Object} packages grouped by services
  */
 export const getPackageGroupsForLabelPurchase = createSelector(
-	( state, siteId = getSelectedSiteId( state ) ) => {
+	( state, orderId, siteId = getSelectedSiteId( state ) ) => {
 		const form = getPackagesForm( state, siteId );
 		if ( ! form || ! form.predefinedSchema || ! form.packages ) {
 			return null;
 		}
 
+		const shippingLabelForm = getForm( state, orderId, siteId );
+		const originCountryCode = getAddressValues( shippingLabelForm.origin ).country;
+		const destinationCountryCode = getAddressValues( shippingLabelForm.destination ).country;
+		const isInternationalOrder =
```

Comment:
What I did on an unrelated server change, to prepare this logic for when we open for other countries, was this:
```js
const isDomestic = originCountry === destinationCountry ||
    ( ACCEPTED_USPS_ORIGIN_COUNTRIES.includes( originCountry ) && ACCEPTED_USPS_ORIGIN_COUNTRIES.includes( destinationCountry ) );
```

So, every reasonable case would be covered:
* Canada -> Canada: Domestic
* Canada -> UK: International
* Canada -> US: International
* US -> Puerto Rico: Domestic
* etc

It doesn't matter right now, but it would be one less thing to think about when/if we open labels to other carriers.